src/content/docs/apm/agents/python-agent/installation/python-agent-admin-script-advanced-usage.mdx

```diff
@@ -25,26 +25,28 @@ This document gives more in-depth detail about these uses of the admin script:
 
 To execute subcommands, provide the name of the subcommand as the first argument to the `newrelic-admin` script. Then add any options after the name of the subcommand. For example:
 
-```
+```shell
 newrelic-admin help
 ```
 
 ## help &lt;command> [#help]
 
 When executed with no further arguments, this provides a listing of all accepted subcommands:
 
-```
-$ newrelic-admin help
-    Usage: newrelic-admin command [options]
+```shell
+newrelic-admin help
```

Comment:
@brnhensley when I run this locally, I get this output instead:

```
Usage: newrelic-admin command [options]

Type 'newrelic-admin help <command>'for help on a specific command.

Available commands are:
  generate-config
  license-key
  local-config
  network-config
  record-deploy
  run-program
  run-python
  server-config
  validate-config
```

Reply:
Hey @rhetoric101 the `[output]` in this is basically so the formatting of line 1 is an input field with a `$` but the following lines are the output, not to be typed in the console and lack the `$` icon. So this was the expected output:

<img width="674" alt="image" src="https://github.com/newrelic/docs-website/assets/48165493/c17a3d44-4f19-49e2-81b6-edf649f9d557">

I did overlook that the blank output line would still be formatted as input with a `$` so I've corrected that with the latest commit.

I also ran this locally and got the same output as you:
```sh
brnhensley % newrelic-admin help
Usage: newrelic-admin command [options]

Type 'newrelic-admin help <command>'for help on a specific command.

Available commands are:
  generate-config
  license-key
  local-config
  network-config
  record-deploy
  run-program
  run-python
  server-config
  validate-config
```

So I've updated that output, and checked the output of the second command as well. 

Does that make sense?